components/eam/bld/configure

```diff
@@ -1390,6 +1385,7 @@ if ($chem_pkg =~ '_mam3') {
     $chem_cppdefs = ' -DMODAL_AERO -DMODAL_AERO_4MODE_MOM ';
 } elsif ($chem_pkg =~ '_mam4' ) {
     $chem_cppdefs = ' -DMODAL_AERO -DMODAL_AERO_4MODE ';
+} elsif ($chem_pkg =~ '_mam4' ) {
```

Comment:
This does not look like it belongs, identical to the elsif directly above but with an empty code block.